base_external/rootfs_overlay/etc/init.d/S98lddmodules.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh
+
+case "$1" in
+    start)
+        modprobe hello.ko
+        
+        /usr/bin/module_load faulty
+        /usr/bin/scull_load
```

Comment:
@swapnilghonge I don't think this will work and this will result in the failure you see in your actions runner at https://github.com/cu-ecen-aeld/assignment-5-swapnilghonge/actions/runs/3269321785/jobs/5387297597#step:6:29437

```
2022-10-18T15:52:27.8803501Z insmod: can't insert './faulty.ko': No such file or directory
2022-10-18T15:52:28.1864922Z insmod: can't insert './scull.ko': No such file or directory
```

You need to customize the module load scripts and/or use modprobe for these.